doc/puppetfile.mkd

```diff
@@ -130,7 +130,8 @@ with a git commit, branch reference, or a tag.
 When a module is installed using `:tag` or `:commit`, r10k assumes that the
 given object is a tag or commit and can do some optimizations around fetching
 the object. If the tag or commit is already available r10k will skip network
-operations when updating the repo, which can speed up install times.
+operations when updating the repo, which can speed up install times. When
+`:commit` is set to track `HEAD`, it will synchronize the module on each run.
```

Comment:
The sync will happen regardless of what key you use (`:ref`, `:commit`, `:branch`, etc.), right?
```suggestion
`:ref` is set to track `HEAD`, it will synchronize the module on each run.
```

Reply:
True. All of `:ref`, `:commit`, and `:branch` are really just syntactic sugar for the same thing these days.